### PR TITLE
Full-width IPs table

### PR DESCRIPTION
**Pull request by kjdchapman**
image

(Also as a bonus tweak, this PR limits the length of service e-mails so that table doesn't look bad)

### CODE REVIEW

File: app/views/ips/_table.html.erb [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-    </tr>
-  </thead>
+  <caption class="govuk-table__caption">
+    <%= location.full_address %>
+  </caption>
   <tbody class="govuk-table__body" id="ips-table">
     <% location.ips.each do |ip| %>
       <tr class="govuk-table__row">
-        <td class="govuk-table__cell half-column" scope="row"><%= ip.address %></td>
+        <td class="govuk-table__cell""><%= ip.address %></td>
```
Comment (emileswarts) [inline app/views/ips/_table.html.erb]: Rogue quote alert

File: app/views/ips/_table.html.erb [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
         <% if ip.available? %>
-          <td class="govuk-table__cell half-column">Available</td>
+          <td class="govuk-table__cell">Available</td>
         <% else %>
-          <td class="govuk-table__cell half-column unavailable-ip">Not available until 6am tomorrow
+          <td class="govuk-table__cell unavailable-ip">Not available until 6am tomorrow
           </td>
         <% end %>
-        <td class="govuk-table__cell half-column"><%= link_to 'Logs', logs_path(ip: ip.address) %></td>
+        <td class="govuk-table__cell"><%= link_to 'View activity', logs_path(ip: ip.address) %></td>
```
Comment (emileswarts) [inline app/views/ips/_table.html.erb]: In my mind "Logs" is still more descriptive than "View activity"

Comment (kjdchapman) [reply app/views/ips/_table.html.erb]: Mm, I think I should have tried "view logs" and got the best of both worlds

File: app/views/ips/_table.html.erb (hunk 1 above)
Comment (kjdchapman) [reply app/views/ips/_table.html.erb]: Sneaky quotes

**Pull request merged**